Makefile

```diff
@@ -89,6 +92,12 @@ install-fetch-config:
 	    && chmod +x fetch_config.rb \
 	    || true
 
+install-terrafile: ## Install terrafile to manage terraform modules
```

Comment:
Can you replace this with the simplified version? https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/Makefile#L30

Reply:
removed